app.js

```diff
@@ -0,0 +1,56 @@
+import express from 'express';
```

Comment:
In general better split up entities/routers by their logic e.g. we can create folder `controllers` with following content:
├──controllers
    ├──cities.js
    ├──index.js
and `cities.js` will be separate router that we can import and simply use in `app.js`:
```suggestion
import express from 'express';
import controllers from './controllers';
import { mwBuilder, errorMiddleware } from './middlewares';

const app = express();
mwBuilder({ app });
app.use('/', controllers);
app.use(errorMiddleware);
export default app;
```